docs/firstobject.md

```diff
@@ -76,78 +76,84 @@ There is of course a lot more to Nodes, Content Types, Formatters, Widgets and i
 
-There are many ways of answering that, archipelago works under the idea of an (or many) Open Schema(s), and that notion permeates the whole environment. Practical answers, simplest way to explain this based on this demo is:
+There are many ways of answering that, Archipelago works under the idea of an (or many) Open Schema(s), and that notion permeates the whole environment. Practical answer and simplest way to explain based on this demo is:
 
-- The `Digital Object` is a generic container for any shape of metadata. Metadata is generated either via this Webform based widget, manually (no worries, power-user need only) or via APIs and can take because of that any shape to express your needs of Digital Objects. You really don't need many types of Digital Objects (trust us!). If you ever need them, those are easy to setup.
-- The Strawberry field Field Widget allows you to attach any `Webform`, built using the [`Webform Module`](https://www.drupal.org/docs/8/modules/webform) and Webforms can be setup in almost infinite ways. Any field, combo, style can be used. Multi Step, single step. We made sure they always only touch/modify data they know how to touch, so even a single input element webform would ensure any previous metadata to persist even if not readable by itself (See the potential?). And Each Webform can be also quite smart!
+- The `Digital Object` is a generic container for any shape of metadata. Metadata is generated either via this Webform-based widget you're currently using, manually (power-user need only) or via APIs. Because of this, Metadata can take any shape to express your needs of Digital Objects and therefore we do not recommend making multiple Digital Object types. However, if you ever do need more Digital Object types, the option is available.
+- The Strawberry field Field Widget allows you to attach any `Webform`, built using the [`Webform Module`](https://www.drupal.org/docs/8/modules/webform) and Webforms can be setup in almost infinite ways. Any field, combo, or style can be used. Multi Step, single step - we made sure they always only touch/modify data they know how to touch, so even a single input element webform would ensure any previous metadata to persist even if not readable by itself (See the potential?). And Each Webform can be also quite smart!
 - The Strawberry field Field Widget will take all your Webform input, process any uploaded files, generate a JSON representation, enrich and complement it with Archipelago specific data and save it for you inside the `Strawberry field`.
 
-We will come back to this later. 
+We will come back to this later.
 
 </div>
 </details>
 
-Make sure you select `Photograph` as `Media Type` and all the fields with a red `*` are filled up. Press `Next: Linked data`. That will load the next step inline. 
```

Comment:
Not sure why this was removed? Both suggestion of what media type to use and the fact they should not press save or preview is quite important to allow a seamless/replicable experience. E.g, repeat the steps by changing the media type and selecting anything else. Result will vary.

Reply:
This explanation was moved up to line 91, right before the screenshot of the Webform.

Reply:
Not the same: `All fields with a red `*` are required fields. Be sure to enter a `Title`, `Media Type`, `Description`, and `Location, Place or Address`. When you are finished, press `Move on to next step` at the bottom of the form.` Implies not that Photograph needs to be selected, it leaves some space for selecting any other type that could end in a different display. It also does not disencourage pressing save/preview. That could be Ok? If preview is disabled (can't remember).

Reply:
@DiegoPino preview is disabled until "Attach Files" (step 4)

Reply:
I mean the Node Preview, not the webform one. Not sure if we disabled that one or not, i really hope we did

Reply:
@mitchellkeaney still unaddressed the fact that people need to select 'Photograph' as media type for this tutorial. Can't merge until that is resolved because its quite important to ensure what they see is what you are providing as screenshot.